profile.md

```diff
@@ -51,7 +51,7 @@ the [Open Service Broker API specification](spec.md) SHALL take precedence.
   interact with Service Brokers to provision, and manage, Service Instances
   and Service Bindings.
 
-- *Service Instance*: An instantiation of a Service offering.
+- *Service Instance*: An instantiation of a Service Offering.
```

Comment:
I'd expect 'Service Offering' then to be defined somewhere? Maybe capitalize on L45 as well then?

Reply:
It is defined in the spec and the terms defined in profile are now pointed to the spec.